README.md

```diff
@@ -2,6 +2,8 @@
 Homebrew Casks for Apples fonts for easy installation.
 All Casks just link to the downloads from [developer.apple.com/fonts](https://developer.apple.com/fonts/).
 
+> All fonts in this tap are also available in [`homebrew/cask-fonts`](https://github.com/Homebrew/homebrew-cask-fonts).
+
```

Comment:
```suggestion
⚠️ This tap is no longer maintained since Apple fonts are now available in the official [`homebrew/cask-fonts`](https://github.com/Homebrew/homebrew-cask-fonts) tap. ⚠️

Homebrew Casks for Apples fonts for easy installation.
All Casks just link to the downloads from [developer.apple.com/fonts](https://developer.apple.com/fonts/).
```